frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -143,91 +147,123 @@ const Uploads = () => {
     <main className="prime-home">
       <div className="grid-container">
         <div className="grid-row">
-          <div className="prime-container">
+          <div className="prime-container card-container">
             <div className="usa-card__header">
-              <h2>COVID-19 CSV Uploads</h2>
+              <h2>Upload your results</h2>
             </div>
-            {reportId && (
-              <div>
-                <div className="usa-alert usa-alert--success">
-                  <div className="usa-alert__body">
-                    <h4 className="usa-alert__heading">
-                      Success: File Accepted
-                    </h4>
-                    <p className="usa-alert__text">
-                      Your file has been successfully transmitted to the
-                      department of health
+            <div className="usa-card__body">
+              <p>
+                Add and report results in bulk using a comma-separated values
+                (CSV) spreadsheet. To upload your spreadsheet:
+              </p>
+              <ul className="usa-list">
+                <li>
+                  <a href="https://reportstream.cdc.gov/assets/csv/ReportStream-StandardCSV-ExampleData-20220509.csv">
+                    Download the spreadsheet template
```

Comment:
```suggestion
                    Download the spreadsheet template and open it in Excel or another spreadsheet application
```
Folding in the gray text currently above the upload feature